rerun_py/rerun_sdk/rerun/log/bounding_box.py

```diff
@@ -136,3 +145,120 @@ def log_obb(
     # Always the primary component last so range-based queries will include the other data. See(#1215)
     if instanced:
         bindings.log_arrow_msg(entity_path, components=instanced, timeless=timeless, recording=recording)
+
+
+@log_decorator
+def log_obbs(
+    entity_path: str,
+    *,
+    half_sizes: npt.ArrayLike | None,
+    positions: npt.ArrayLike | None = None,
+    rotations_q: npt.ArrayLike | None = None,
+    colors: Color | Colors | None = None,
+    stroke_widths: npt.ArrayLike | None = None,
+    labels: Sequence[str] | None = None,
+    class_ids: OptionalClassIds | None = None,
+    ext: dict[str, Any] | None = None,
+    timeless: bool = False,
+    recording: RecordingStream | None = None,
+) -> None:
+    """
+    Log a 3D Oriented Bounding Box, or OBB.
+
+    Example:
+    --------
+    ```
+    rr.log_obb("my_obb", half_size=[1.0, 2.0, 3.0], position=[0, 0, 0], rotation_q=[0, 0, 0, 1])
+    ```
+
+    Parameters
+    ----------
+    entity_path:
+        The path to the oriented bounding box in the space hierarchy.
+    half_sizes:
+        Nx3 Array. Each row is the [x, y, z] half dimensions of an OBB.
+    positions:
+        Optional Nx3 array. Each row is [x, y, z] positions of an OBB in world space.
+    rotations_q:
+        Optional Nx3 array. Each row is quaternion coordinates [x, y, z, w] for the rotation from model to world space.
+    colors:
+        Optional Nx3 or Nx4 array. Each row is RGB or RGBA in sRGB gamma-space as either 0-1 floats or 0-255 integers,
+        with separate alpha.
+    stroke_widths:
+        Optional array of the width of the line edges.
+    labels:
+        Optional array of text labels placed at `position`.
+    class_ids:
+        Optional array of class id for the OBBs.  The class id provides colors and labels if not specified explicitly.
+    ext:
+        Optional dictionary of extension components. See [rerun.log_extension_components][]
+    timeless:
+        If true, the bounding box will be timeless (default: False).
+    recording:
+        Specifies the [`rerun.RecordingStream`][] to use.
+        If left unspecified, defaults to the current active data recording, if there is one.
+        See also: [`rerun.init`][], [`rerun.set_global_data_recording`][].
+
+    """
+    recording = RecordingStream.to_native(recording)
+
+    colors = _normalize_colors(colors)
+    radii = _normalize_radii(stroke_widths)
+    radii / 2
```

Comment:
this keeps coming up, I thought we fixed this at some point in the backend?

Reply:
Backend just uses radii. But our API uses stroke_width, hence the divide by 2. I should have renamed the var above though.